app/experimenter/nimbus-ui/src/components/TableHighlights/index.tsx

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import { AnalysisData } from "../../lib/visualization/types";
+import {
+  HIGHLIGHTS_METRICS_LIST,
+  METRICS_TIPS,
+  METRIC,
+  BRANCH_COMPARISON,
+  SEGMENT_TIPS,
+  TABLE_LABEL,
+} from "../../lib/visualization/constants";
+import { getTableDisplayType } from "../../lib/visualization/utils";
+import { getExperiment_experimentBySlug_featureConfig } from "../../types/getExperiment";
+import TableVisualizationRow from "../TableVisualizationRow";
+import { ReactComponent as Info } from "../../images/info.svg";
+
+type TableHighlightsProps = {
+  featureConfig: getExperiment_experimentBySlug_featureConfig;
+  results: AnalysisData["overall"];
+};
+
+const TableHighlights = ({
+  featureConfig,
+  results = {},
+}: TableHighlightsProps) => {
+  const highlightMetricsList = [
+    {
+      value: `${featureConfig.slug.replace(/-/g, "_")}_ever_used`,
```

Comment:
I just realized something else here. We should be looking at primary probe sets not `featureConfig` in this case. This will need to be changed

Reply:
Also, It looks like this only adds 1 featureConfig. When this is switched to primary probe sets, there could be multiple primary probe sets and it needs to add all of them

Reply:
@emtwo Ohh I see, in `rapid` the highlights table uses `experimentData.features`, which I must have mistakenly thought was `featureConfig` in nimbus, of which there can only be one type of. I thought the primary/secondary probe sets would be displayed in the primary/secondary metric tables. I'll go take a look 🤔

Reply:
Updated!